### PR TITLE
docs: DOC-1166 image update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the image and touches up the language for the Order of Operations page.

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-2773--docs-spectrocloud.netlify.app/architecture/orchestration-spectrocloud/)

## Jira Tickets

🎫 [DOC-1166](https://spectrocloud.atlassian.net/browse/DOC-1166)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1166]: https://spectrocloud.atlassian.net/browse/DOC-1166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/architecture/orchestration-spectrocloud.md [hunk 1]
```diff
@@ -1,47 +1,56 @@
...
-   worker's MachineDeployment or VSphereMachineTemplate. Cluster API running in CP-A will provision the remaining
-   control plane and worker nodes.
+5. CP-A agent will retrieve the required specifications for the cluster, such as the number of nodes. CP-A will generate
+   and update the remaining CAPI resources, e.g: update replicas to 3 for KubeadmControlPlane, create the worker's
+   MachineDeployment or VSphereMachineTemplate. The Cluster API active in CP-A will provision the remaining control
+   plane and worker nodes.
 
-6. The Palette agent will install all the additional add-ons as specified by the cluster's cluster profile (e.g:
-   logging, monitoring, security).
+6. The Palette agent will install all the additional add-ons as the cluster cluster profile specifies.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/architecture/orchestration-spectrocloud.md:53]:
```suggestion
6. The Palette agent will install all the additional add-ons as the cluster profile specifies.
```

File: docs/docs-content/architecture/orchestration-spectrocloud.md [hunk 2]
```diff
@@ -56,22 +65,23 @@ Palette's decentralized model is based on a "decentralized management - local po
...
-The target workload cluster is then responsible for provisioning and maintaining the remaining control-plane and worker
-nodes. All Day-2 operations which result in node changes, including OS/K8s upgrades, scaling, and K8s certificate
-rotation, are triggered by changes to the Cluster API resources in the target workload cluster.
+The target workload cluster is responsible for provisioning and maintaining the remaining control plane and worker
+nodes. All Day-2 operations, which may result in node changes, including the operating system and Kubernetes upgrades,
+scaling, and Kubernetes SSL certificate rotation, are triggered by changes to Cluster API resources in the target
+workload cluster.
 
-Palette pivots these clusters for several reasons, related to scalability and availability:
+Palette pivots these clusters for reasons such as:
```
Comment (caroldelwing) [inline docs/docs-content/architecture/orchestration-spectrocloud.md:77]: This sentence is a bit confusing here. We were just talking about the workload cluster, now we are talking about pivot again. I'm not sure if I understood :)

**Review by caroldelwing — APPROVED** [review]
Approved with a few comments :)

File: docs/docs-content/architecture/orchestration-spectrocloud.md [hunk 3]
```diff
@@ -1,47 +1,56 @@
...
+Check out the [Cluster API concepts](https://cluster-api.sigs.k8s.io/user/concepts) page for a detailed explanation of
+the components and concepts used in the Cluster API.
 
-Cluster API demonstratively manages the lifecycle of a cluster (creation, scaling, enhancement, and deletion) and helps
-in automating the process of cluster lifecycle management for platform operations. Cluster API also helps in consistent
-and repeated cluster deployment across multiple infrastructure environments.
+:::
 
-<br />
+Cluster API manages the lifecycle of a cluster. It helps in automating the process of cluster lifecycle management for
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/architecture/orchestration-spectrocloud.md:23]: @caroldelwing Cluster API is a separate concept, not to be confused with the Kubernetes API server inside a cluster.

File: docs/docs-content/architecture/orchestration-spectrocloud.md (hunk 2 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/architecture/orchestration-spectrocloud.md:77]: @caroldelwing, great catch. Let me take a stab at updating the old language and making it clearer.

File: docs/docs-content/architecture/orchestration-spectrocloud.md [hunk 4]
```diff
@@ -1,47 +1,56 @@
...
+platform operations. The Cluster API also helps ensure a consistent experience in cluster deployment across multiple
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/architecture/orchestration-spectrocloud.md:24]: Accepted but with a minor change.

**Pull request merged**